README.md

```diff
@@ -96,3 +97,4 @@ Pro tip: watch this repository to get notified about new plugins.
 - [tmux-yank](https://github.com/tmux-plugins/tmux-yank) - Plugin for copying
   to system clipboard. Works on MacOS, Linux and Cygwin.
 - [tpm](https://github.com/tmux-plugins/tpm) - Tmux plugin manager.
+
```

Comment:
Can you please remove this last line? Everything else is ok and I'll merge afterwards.

Reply:
My bad, I'll fix that, thank you